### PR TITLE
Added Ford Fulkerson Method in Javascript

### LINKED ISSUE #2477: Ford Fulkerson Method
- [x] C
- [x] C++
- [ ] C# - @shreyakapoor08
- [ ] CoffeeScript
- [ ] Dart - @spacemonkey1101
- [ ] Go
- [x] Java
- [x] JS - @bohemian98
- [ ] Kotlin
- [ ] Php - @mharshita
- [x] Python
- [x] Readme - @syedareehaquasar
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by bohemian98**
Fixes #2477

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Javascript code for Ford Fulkerson Method

#### Languages Used:
- Javascript

#### Files Added:
- Ford_Fulkerson_Method.js

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by bohemian98** [comment]
@jainaman224 @MastersAbh Please review my PR.

**Comment by singh-shreya6** [comment]
Working on it?

**Comment by bohemian98** [comment]
yes @singh-shreya6

**Comment by bohemian98** [comment]
@singh-shreya6 I have made the changes and squashed my commits. Please review :)

File: Ford_Fulkerson_Method/Ford_Fulkerson_Method.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    for(var j=0;j<V;j++)
+    {
+        console.log(`Enter the weight form ` , i , `to `,j, `: `);
+        var a=readlineSync.question();
+        graph[i][j]=a;
+    }
+}
+var source=readlineSync.question("Enter source: ");
+var sink=readlineSync.question("Enter Sink: ");
+var ans=fordFulkerson(graph,source,sink,V);
```
Comment (singh-shreya6) [inline Ford_Fulkerson_Method/Ford_Fulkerson_Method.js:115]: Space around = and after comma

File: Ford_Fulkerson_Method/Ford_Fulkerson_Method.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+console.log('Maximum flow from source to sink is: ',ans);
```
Comment (singh-shreya6) [inline Ford_Fulkerson_Method/Ford_Fulkerson_Method.js:116]: Space after comma

**Comment by bohemian98** [comment]
@singh-shreya6 Please review. Hopefully there are no more changes to make but if there are any please let me know. I would love to resolve those :). Thanks.

**Pull request merged**